token-syncer/src/token_syncer/commands/syncer.clj

```diff
@@ -75,7 +75,7 @@
   "Syncs a given token description on all clusters.
    If the cluster-url->token-data says that a given token was not successfully loaded, it is skipped.
    Token sync-ing is also skipped if the tokens are active and the roots are different."
-  [{:keys [store-token]} cluster-urls token latest-token-description cluster-url->token-data]
+  [{:keys [store-token]} cluster-urls token latest-token-description metadata-name cluster-url->token-data]
```

Comment:
Would it make sense to rename `metadata-name` to `opt-out-metadata-name`? I think that would make it more immediately clear what that metadata key-value pair is being used for.

Reply:
Renamed to `opt-out-metadata-name`